src/tests/Details.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import Details from '../components/Details';
+
+describe('Details Component test', () => {
+  it('Renders the details of city weather data', () => {
+    const app = renderer
+      .create(
+        <Provider store={store}>
+          <Details />
+        </Provider>,
+      )
+      .toJSON();
+    expect(app).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] This test is currently failing. I will suggest you look at the error closely and fix it.

![image](https://user-images.githubusercontent.com/51296741/149642450-87bb0096-a5d5-49f5-9984-d02ba0fe47fc.png)

- [ ] I will suggest you add more tests by testing more implemented features. Currently, we only have 4 tests cases.